src/navStacks/HomeStack/index.tsx

```diff
@@ -61,111 +64,115 @@ const AuthStack = createNativeStackNavigator<any>();
 //
 function AuthNavs() {
   return (
+
     <AuthStack.Navigator
       screenOptions={{ headerShown: false }}
       initialRouteName="LoginScreen"
     >
       <AuthStack.Screen name="LoginScreen" component={Login} />
+      <AuthStack.Screen name="Register" component={Register} />
+      <AuthStack.Screen name="VerificationCode" component={VerificationCode} />
+      <AuthStack.Screen name="Home" component={Home} />
```

Comment:
We don't need to add Home to the AuthStack, it should already exist in the `MainBottomNavScreens`. We can just route the user to MainBottomNavScreens via `navigator.navigate('MainStack')` The MainStack has a prop to determine the default route, and we can add other logic in there too as needed.


Here is a crude representation of how nav stacks and tab navs are combined. A nav stack is a linear set of routes. A tab navigator adds a special bottom bar with icons and whatnot to route you around the app. Each of those tab navigators though can either be a single page or their own set of sub pages (a stack nav)
<img width="594" alt="image" src="https://github.com/eklemen/CraftPile/assets/10147000/6323656e-d07e-46ce-97fb-33a618c545ac">